<launch>
  <arg name="language" default="en"/>
  <arg name="intern_file" default="none"/>
  <arg name="files" default="
  $(find ontologenius)/files/attribute.owl
  $(find ontologenius)/files/positionProperty.owl
  $(find ontologenius_query)/files/testIndividuals.owl
  "/>
  <node name="ontologenius_core" pkg="ontologenius" type="ontoloGenius" output="screen" args="$(arg language) $(arg intern_file) $(arg files)"> </node>
  <node name="ontologenius_rest" pkg="ontologenius" type="REST.py" output="screen"> </node>
  <node name="ontologenius_query" pkg="ontologenius_query" type="ontologenius_query" output="screen"> </node>
</launch>
